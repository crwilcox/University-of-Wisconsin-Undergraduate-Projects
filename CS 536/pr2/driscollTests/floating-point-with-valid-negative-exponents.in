123456789.123456789e-123
738.123e-45
82973.322456e-1
743.22e-45
