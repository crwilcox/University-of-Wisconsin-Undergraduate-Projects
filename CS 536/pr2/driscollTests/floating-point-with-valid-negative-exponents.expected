1:1:FLOAT_LIT: 1.2345678912345679E-115
2:1:FLOAT_LIT: 7.38123E-43
3:1:FLOAT_LIT: 8297.3322456
4:1:FLOAT_LIT: 7.4322E-43
